### DISCUSSION TITLE
Change MTU?

### DISCUSSION THREAD

**Discussion by AstraLuma** [hbldh/bleak#1350] [Q&A] [answered]
One of the features of GATT is being able to request a different MTU. I see this in other BLE APIs, but not bleak?

Am I missing something or is this a missing feature?

**Comment by dlech** [comment, marked as answer]
You did not miss anything. Android is the only OS that provides APIs to specify the MTU. 

If you have control over the device firmware, I would suggest to have it specify the MTU.

**Reply by AstraLuma** [reply]
and Python on mobile is pretty cutting edge. ok, thanks!
